tests/bsim/bluetooth/mesh/src/test_blob.c

```diff
@@ -927,7 +929,7 @@ static void test_cli_trans_complete(void)
 	blob_cli_xfer.xfer.size = CONFIG_BT_MESH_BLOB_BLOCK_SIZE_MIN * 4;
 	blob_cli_xfer.xfer.id = 1;
 	blob_cli_xfer.xfer.block_size_log = 9;
-	blob_cli_xfer.xfer.chunk_size = 377;
+	blob_cli_xfer.xfer.chunk_size = MAX_CHUNK_SIZE;
```

Comment:
Please revert changes in this file. Test was especially designed to use exact digits to cover splitting block on chunks and chunks on segments to cover full functionality at maximum. Also, the dfu and blob tests were optimized to reduce CI time due to Zephyr's community request. The current settings are a tradeoff between test coverage and CI loading.
Please take a look at this PR: #71683